.github/workflows/unittests.yml

```diff
@@ -10,14 +10,25 @@
  jobs:
    unittests:
      name: CI (${{ matrix.os }}-${{ matrix.environment-file }})
+     env:
+       INSTALL: pip install -e . --no-deps --force-reinstall
```

Comment:
Why is `--no-deps --force-reinstall` needed here?

Reply:
Not sure. Was just there before. Let's try without?

Reply:
Tests appear to be passing w/o `--no-deps --force-reinstall`.

Reply:
It should make no difference if the env is properly set up.